linux_os/guide/system/auditing/auditd_configure_rules/audit_execution_selinux_commands/audit_rules_execution_seunshare/rule.yml

```diff
@@ -0,0 +1,40 @@
+documentation_complete: true
+
+prodtype: rhel7,fedora
+
+title: 'Record Any Attempts to Run seunshare'
+
+description: |-
+    At a minimum, the audit system should collect any execution attempt
+    of the <tt>seunshare</tt> command for all users and root. If the <tt>auditd</tt>
+    daemon is configured to use the <tt>augenrules</tt> program to read audit rules
+    during daemon startup (the default), add the following lines to a file with suffix
+    <tt>.rules</tt> in the directory <tt>/etc/audit/rules.d</tt>:
+    <pre>-a always,exit -F path=/usr/sbin/seunshare -F perm=x -F auid&gt;=1000 -F auid!=4294967295 -F key=privileged-priv_change</pre>
```

Comment:
use `unset` instead of 4294967295 please. Using numeric value is error-prone. (won't repeat for every instance, but please change it everywhere).

Reply:
The tests were failing of Fedora because the audit rules for privileged commands were hardcoded for `rhel7`.
I have added short script that generates necessary rules to setup pass scenarios.